gprofiler/client.py

```diff
@@ -73,11 +83,13 @@ def _send_request(
                 json.dump(data, gzip_file, ensure_ascii=False)  # type: ignore
             opts["data"] = buffer.getvalue()
 
+        opts["params"] = self._get_query_params() + [(k, v) for k, v in params.items()]
+
         resp = self._session.request(method, "{}/{}".format(self.get_base_url(api_version), path), **opts)
         if 400 <= resp.status_code < 500:
             try:
-                data = resp.json()
-                raise APIError(data["message"], data)
+                resp_data = resp.json()
+                raise APIError(resp_data["message"], resp_data)
```

Comment:
If you already touched this section I suggest  changing `resp_data` -> `response_data`

Reply:
k